desktop/src/com/unciv/app/desktop/PlatformSpecificHelpersDesktop.kt

```diff
@@ -15,4 +17,9 @@ class PlatformSpecificHelpersDesktop : GeneralPlatformSpecificHelpers {
             false
         }
     }
+
+    override fun notifyTurnStarted() {
+        turnNotifier.turnStarted()
```

Comment:
I still expected something along `System.getProperty("os.name")?.startsWith("Windows")` - the `Native.load(User32::class.java)` _might_ `null` out cleanly on *ix - hopefully? The OS flavour distinction is in crash info, and some refactoring to have it in one place might be nice, but I have no obvious picture how. `object` level probably.

Reply:
Imagine on Tux the "Preparing Wine Prefix..." dialog popping up :rofl: